### PR TITLE
Add endpoint to GET a Service config

### PR DESCRIPTION
**Pull request by Kegsay**

### CODE REVIEW

File: src/github.com/matrix-org/go-neb/api.go [hunk 1]
```diff
@@ -228,3 +229,37 @@ func (s *configureServiceHandler) OnIncomingRequest(req *http.Request) (interfac
 		NewConfig types.Service
 	}{body.ID, body.Type, oldService, service}, nil
 }
+
+type getServiceHandler struct {
+	db *database.ServiceDB
+}
+
+func (h *getServiceHandler) OnIncomingRequest(req *http.Request) (interface{}, *errors.HTTPError) {
+	if req.Method != "GET" {
```
Comment (NegativeMjark) [inline src/github.com/matrix-org/go-neb/api.go]: GET with a body is probably a bad idea.

Comment (NegativeMjark) [inline src/github.com/matrix-org/go-neb/api.go]: The most obvious objection being that the go http client doesn't support without building the request by hand.

Comment (Kegsay) [inline src/github.com/matrix-org/go-neb/api.go]: Oops, my bad.

Comment (Kegsay) [inline src/github.com/matrix-org/go-neb/api.go]: POST now

**Comment by NegativeMjark** [comment]
LGTM

**Pull request merged**